gdal/ogr/ogrsf_frmts/cad/gdalcaddataset.cpp

```diff
@@ -117,15 +117,18 @@ int GDALCADDataset::Open( GDALOpenInfo* poOpenInfo, CADFileIO* pFileIO,
     osCADFilename = pFileIO->GetFilePath();
     SetDescription( poOpenInfo->pszFilename );
 
-    int nMode = atoi(CSLFetchNameValueDef( poOpenInfo->papszOpenOptions, "MODE", 
-                                                                           "2"));
+    const char * papszOpenOptions = CSLFetchNameValueDef( poOpenInfo->papszOpenOptions, "MODE", 
+                                                                           "READ_FAST");
+    const char * papszReadUnsupportedGeoms = CSLFetchNameValueDef( 
```

Comment:
Deliberately not used, since library has incomplete capabilities. (Will be fixed in 1-2 days).